golang.go

```diff
@@ -0,0 +1,7 @@
+package main
```

Comment:
language: Go
<!-- lookout footnote separator -->
_analyzer Dummy, please, [tell us](http://google.es). HERE IS. NOW YES_